test/kubetest2-capi/OWNERS

```diff
@@ -0,0 +1,13 @@
+# See the OWNERS docs at https://go.k8s.io/owners for information on OWNERS files.
+# See the OWNERS_ALIASES file at https://github.com/kubernetes-sigs/cluster-api/blob/master/OWNERS_ALIASES for a list of members for each alias. 
+
+approvers:
+  - cluster-api-admins
+  - cluster-api-maintainers
+  - benmoss
+  - elmiko
+  - randomvariable
```

Comment:
Is this list up to date?

Reply:
i feel like @benmoss may not want to stay on that list. i'm happy to be an approver, but i wonder if we don't have some other folks who would like to get more involved as well?